crates/ruff_linter/src/rules/pyflakes/rules/repeated_keys.rs

```diff
@@ -195,24 +195,24 @@ pub(crate) fn repeated_keys(checker: &mut Checker, dict: &ast::ExprDict) {
                         },
                         key.range(),
                     );
-                    let comparable_value: ComparableExpr = (&dict.values[i]).into();
+                    let comparable_value: ComparableExpr = (&dict.items[i].value).into();
```

Comment:
I wanted to do this here:

```suggestion
                    let comparable_value: ComparableExpr = (&dict.values()[i]).into();
```

But the borrow checker wouldn't let me